src/Compilers/CSharp/Test/Semantic/Semantics/ObjectAndCollectionInitializerTests.cs

```diff
@@ -2923,13 +2935,14 @@ static void Main()
 }
 ";
             string expectedOperationTree = @"
-IObjectCreationExpression (Constructor: C..ctor()) (OperationKind.ObjectCreationExpression, Type: C, IsInvalid) (Syntax: 'new C { 1 }')
```

Comment:
I would expect to still get IObjectCreationExpression node as the root.

Reply:
done